### PR TITLE
Add Tizen 5.0 RID

### PR DESCRIPTION
**Pull request by gbalykov**
cc @alpencolt @Dmitri-Botcharnikov @eerhardt 

related comment: https://github.com/dotnet/core-setup/pull/4307#issuecomment-401823214

### CODE REVIEW

**Comment by eerhardt** [comment]
@dotnet-bot test this please

File: src/pkg/projects/netcoreappRIDs.props [hunk 1]
```diff
@@ -33,5 +33,8 @@
     <UnofficialBuildRID Include="tizen.4.0.0-armel">
```
Comment (eerhardt) [inline src/pkg/projects/netcoreappRIDs.props]: Are both of these RIDs going to be supported going forward? Or another way of asking: Are you going to produce `runtime` packages for both `tizen.4.0.0` and `tizen.5.0.0` for every build of master?

Comment (gbalykov) [reply src/pkg/projects/netcoreappRIDs.props]: @eerhardt, yes, we'll use both RIDs

**Comment by eerhardt** [comment]
It doesn't appear that this fixed the Tizen leg.  It is still failing.

The reason it is failing is because of errors like this:

```
Unable to find package transport.runtime.tizen.4.0.0-armel.Microsoft.NETCore.Runtime.CoreCLR with version (>= 3.0.0-preview1-26705-05)

Found 124 version(s) in https://tizen.myget.org/F/dotnet-core/api/v3/index.json [ Nearest version: 3.0.0-preview1-26624-03 ]
```

It looks like there hasn't been a `runtime.tizen.4.0.0` build of coreclr in 2 weeks.  Looking on https://tizen.myget.org/feed/dotnet-core/package/nuget/transport.runtime.tizen.4.0.0-armel.Microsoft.NETCore.Runtime.CoreCLR, I see the last version was built on `Sun, 24 Jun 2018 12:49:12 GMT`

**Comment by gbalykov** [comment]
@eerhardt we currently are unable to build nuget packages for `tizen.5.0.0-armel` as https://github.com/dotnet/corefx/pull/30640 is not merged yet and some 2.1 packages, which are used during CoreFX build, do not contain Tizen 5.0 RID. So CI for both CoreFX and Core-Setup will be fixed only after the mentioned pull request is merged

**Comment by eerhardt** [comment]
From my understanding this failure has nothing to do with the `tizen.5.0.0-armel` RID.  It is failing to find `tizen.4.0.0-armel` packages.  Why aren't the `4.0` packages being built?

**Comment by gbalykov** [comment]
@eerhardt the problem is that CI build should depend only on `tizen.5.0.0-armel`, however, due to missing RIDs, `tizen.4.0.0-armel` dependency also appears. `tizen.4.0.0-armel` builds will be reenabled after `tizen.5.0.0-armel` build is fully fixed

**Comment by gbalykov** [comment]
@eerhardt actually, my last message considered #4246, as they are related

**Comment by gbalykov** [comment]
@eerhardt could this be merged? 
Related PR for CoreFX: https://github.com/dotnet/corefx/pull/30875

**Comment by eerhardt** [comment]
Yes, I can merge it. I'd like to get the Tizen leg green again.  Do we know all the steps necessary to get it passing?

**Comment by gbalykov** [comment]
@eerhardt, I have some problems during CoreFX/Core-Setup builds.

1. I see next situation during `release/2.1` build of CoreFX:

next files contain only `tizen.4.0.0` RID references, and no `tizen.5.0.0` references at all

```
./Tools/dotnetcli/shared/Microsoft.AspNetCore.All/2.1.0-rc1-final/Microsoft.AspNetCore.All.deps.json
./Tools/dotnetcli/shared/Microsoft.NETCore.App/2.1.0-rc1/Microsoft.NETCore.App.deps.json
./Tools/dotnetcli/shared/Microsoft.AspNetCore.App/2.1.0-rc1-final/Microsoft.AspNetCore.App.deps.json

./packages/microsoft.netcore.ilasm/2.1.3-servicing-26708-02/runtime.json
./packages/microsoft.netcore.dotnethostpolicy/2.1.3-servicing-26708-02/runtime.json
./packages/microsoft.netcore.dotnethost/2.1.3-servicing-26708-02/runtime.json
./packages/microsoft.netcore.runtime.coreclr/2.1.3-servicing-26708-02/runtime.json
./packages/microsoft.netcore.testhost/2.1.3-servicing-26708-02/runtime.json
./packages/microsoft.netcore.dotnethostresolver/2.1.3-servicing-26708-02/runtime.json
./packages/microsoft.netcore.dotnetapphost/2.1.3-servicing-26708-02/runtime.json
./packages/microsoft.netcore.app/2.1.0-rc1/runtime.json
./packages/microsoft.netcore.jit/2.1.3-servicing-26708-02/runtime.json
```

this leads to `tizen.4.0.0` dependency, which is added to `./bin/obj/Unix.AnyCPU.Release/runtime/netcoreapp/Unix-armel/project.assets.json`, and `tizen.4.0.0` packages are used later.

2. Besides, `./bin/obj/Unix.AnyCPU.Release/runtime/netcoreapp/Unix-armel/project.assets.json` contains next

```
".NETCoreApp,Version=v2.1/tizen.5.0.0-armel": {
      "Microsoft.NETCore.DotNetAppHost/2.1.3-servicing-26708-02": {
        "type": "package",
        "dependencies": {
          "runtime.tizen.4.0.0-armel.Microsoft.NETCore.DotNetAppHost": "2.1.3-servicing-26708-02"
        }
      }
```

How is this circular dependency CoreFX (NETCoreApp) -> Core-Setup (DotNetAppHost) -> CoreFX (NETCoreApp) could be handled, when there are no nuget packages for new RID (`tizen.5.0.0`) yet?

**Comment by eerhardt** [comment]
@gbalykov - we just got a new 2.1.3 servicing build of core-setup out yesterday: `2.1.3-servicing-26718-02`. I verified that the `Microsoft.NETCore.DotNetAppHost` runtime.json contains a reference to 

```json
    "tizen.5.0.0-armel": {
      "Microsoft.NETCore.DotNetAppHost": {
        "runtime.tizen.5.0.0-armel.Microsoft.NETCore.DotNetAppHost": "2.1.3-servicing-26718-02"
      }
    },
```

So you should be able to move up to that version and build again.

**Pull request merged**